src/git/runner.go

```diff
@@ -180,27 +180,16 @@ func (r *Runner) CommitStagedChanges(message string) error {
 	return nil
 }
 
-// CommitWithMessageAndAuthor .
-func (r *Runner) CommitWithMessageAndAuthor(message, author string) error {
-	_, err := r.Run("git", "commit", "-m", message, "--author", author)
-	if err != nil {
-		return fmt.Errorf("cannot commit with message %q and author %q: %w", message, author, err)
+// Commit commits staged changes with an optional message and author .
```

Comment:
Remove the space before the period.

Reply:
Suggestion to avoid stuttering:

```suggestion
// Commit performs a commit of the staged changes with an optional custom message and author.
```